docs/sdk/lib-ethers.ethersliquity.connect_1.md

```diff
@@ -16,7 +16,7 @@ static connect(signerOrProvider: EthersSigner | EthersProvider, optionalParams?:
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  signerOrProvider | [EthersSigner](./lib-ethers.etherssigner.md) \| [EthersProvider](./lib-ethers.ethersprovider.md) | Ethers <code>Signer</code> or <code>Provider</code> to use for connecting to the Ethereum network. |
+|  signerOrProvider | [EthersSigner](./lib-ethers.etherssigner.md) \| [EthersProvider](./lib-ethers.ethersprovider.md) | Ethers <code>Signer</code> or <code>Provider</code> to use for connecting to the EVM-compatible network. |
```

Comment:
it says "don't edit this file..." in the beginning

Reply:
That's annoying. Is that because it will be overwritten?

Reply:
yes
the docs are auto-generated